SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-only

We do not have the '-E' argument in our version of sed. But it is
functionally equivalent to '-r' (at least for this purpose).

--- scripts/headers_install.shk	2021-06-01 18:48:44.732934584 +1000
+++ scripts/headers_install.sh	2021-06-01 18:48:56.954050654 +1000
@@ -29,7 +29,7 @@
 	exit 1
 fi
 
-sed -E -e '
+sed -r -e '
 	s/([[:space:](])(__user|__force|__iomem)[[:space:]]/\1/g
 	s/__attribute_const__([[:space:]]|$)/\1/g
 	s@^#include <linux/compiler(|_types).h>@@
